getting_started.ipynb

```diff
@@ -0,0 +1,840 @@
+     "text": [
+      "899.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "print spectrum_1.sum()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The value returned by `sum`, should roughly equal:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "900.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "print num_decays * efficiency"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also inspect the raw data structure. This is saved in the `_data` member of the `Spectra` class:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[[ 0.  0.  0. ...,  0.  0.  0.]\n",
+      " [ 0.  0.  0. ...,  0.  0.  0.]\n",
+      " [ 0.  0.  0. ...,  0.  0.  0.]\n",
+      " ..., \n",
+      " [ 0.  0.  0. ...,  0.  0.  0.]\n",
+      " [ 0.  0.  0. ...,  0.  0.  0.]\n",
+      " [ 0.  0.  0. ...,  0.  0.  0.]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print spectrum_1._data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note: you probably won't see any entries in the above. For large arrays, numpy only prints the first three and last three entries. Since our energy range is in the middle, all our events are in the** `...` **part at the moment. But we will see entries printed out later when we apply some cuts.**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Plotting"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Another useful way to inspect the `Spectra` created is to plot it. Support is available within echidna to plot using either `ROOT` or `matplotlib` and there are some useful plotting functions available in the `plot` an `plot_root` modules."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import echidna.output.plot as plot\n",
+    "import echidna.output.plot_root as plot_root"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "To plot the projection of the spectrum on the `energy_mc` axis:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7f8c08958050>"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "plot.plot_projection(spectrum_1, \"energy_mc\", fig_num=1)"
```

Comment:
The plot didn't appear. This is what was written:
<matplotlib.figure.Figure at 0x3942810>